Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -68,6 +68,16 @@
 
 #if PLATFORM(VISION)
 #import "MRUIKitSPI.h"
+
+// Forward declarations for ImmersiveSpace API (visionOS 2.0+)
+// TODO: Remove once these are in MRUIKitSPI.h
```

Comment:
that TODO comment is unlikely to be correct.
Those interfaces aren't SPI

We aren't building for visionOS < 2.0 anymore nor will we ever be going to. Please use the proper include for it, no need for forward declarations